frontend/lib/util/sanitize_contact_details.dart

```diff
@@ -0,0 +1,19 @@
+final _httpRegex = RegExp(r"^https?://");
+
+String sanitizePhoneNumber(String telephone) => telephone
+    .trim() // remove outer whitespace
+    .replaceAll(RegExp(r"\s+"), "-") // replace inner whitespace
+    .replaceAll(RegExp(r'[^\d+-]'), ""); // remove illegal characters
+
+String prepareWebsiteUrlForDisplay(String website) => website
+    .trim()
+    .replaceAll(_httpRegex, "");
+
+String prepareWebsiteUrlForLaunch(String website) {
+  var trimmed = website.trim();
+  if (_httpRegex.hasMatch(trimmed)) {
+    return trimmed;
+  } else {
+    return "https://$trimmed";
```

Comment:
Or should we go with http? I think the security aspect is not that important here.

Reply:
I don't really have a strong opinion here. Unfortunately, I believe that there are still a lot of websites out there with only http, hence I'd suggest to actually do `http`... Good servers will redirect to https anyways